models/leaderboard/getLeaderBoard.js

```diff
@@ -0,0 +1,20 @@
+const { pool } = require('../db');
+/**
+ *
+ * @return {Promise}
+ */
+function getLeaderBoard() {
+  return new Promise((resolve, reject) => {
+    pool.query(
+      `SELECT email,phone,admin,college,name,score FROM users ORDER BY score DESC LIMIT 10`,
```

Comment:
Check out the schema properly bro..Dont just randomly add new columns. Its  not possible to update scores. It will be dynamic